packages/cli/bin/index.js

```diff
@@ -10,13 +10,16 @@ const start = require('../lib/start');
 const devPlugin = require('../lib/devPlugin');
 const dataset = require('../lib/dataset');
 const childProcess = require('child_process');
+const debugLog = require('debug')('cli/bin');
```

Comment:
As for the debug's habit, use `debug` is better and simple:

```suggestion
const debug = require('debug')('cli/bin');
```